specification/operationsmanagement/resource-manager/readme.cli.md

```diff
@@ -0,0 +1,8 @@
+ CLI
```

Comment:
Is this space expected?

Reply:
Thanks Tianyi, just commit a correctness.